### ISSUE TITLE
Should use CFBundleName instead of CFDisplayBundleName for .app name

### ISSUE THREAD

**Issue by LuKePicci** [egramtel/dotnet-bundle#5]
I think this should use `CFBundleName` to determine output .app name, not `CFBundleDisplayName`

https://github.com/egramtel/dotnet-bundle/blob/7e681e61b259222e4e5269a1423f92e7b4649d2d/DotNet.Bundle/StructureBuilder.cs#L18

**Comment by x2bool** [comment]
Why? Is there any reason CFBundleName is better? 

https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundlename

https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundledisplayname

**Comment by LuKePicci** [comment]
Because that is what I read on https://github.com/egramtel/dotnet-bundle/blob/master/README.md:

>    \<CFBundleName>AppName\</CFBundleName> \
>    \<CFBundleDisplayName>App Name\</CFBundleDisplayName>
>    \<CFBundleIdentifier>com.example\</CFBundleIdentifier>
>    ...

And that is what Xcode does (and Visual Studio for Mac as well on Xamarin project templates)

**Comment by LuKePicci** [comment]
Actually it could be even better to have a separate option for setting bundle filename so I can avoid renaming my system extension bundles (which are required to have a filename matching their bundle identifier with `.systemextension` filename extension).

**Comment by x2bool** [comment]
Yes, I think we could go with custom property and fallback to CFBundleName if it's not set.
